randovania/games/fusion/logic_database/header.json

```diff
@@ -1169,6 +1169,9 @@
             2,
             3,
             4
+        ],
+        "WallJump": [
```

Comment:
I'm not sure where this WallJump came from. I did not create this while checking node connections.

Reply:
When the first trick is added at a certain level, it gets added to the header file. This tells it to show up in the preset settings.
![image](https://github.com/JeffGainsNGames/randovania/assets/125271738/c4955725-e410-412f-bf95-36bd2636eb2a)
We can resolve this easily after the other changes are addressed.